snapshot-labs/sx-monorepo
# feat: show share modal after vote

### Summary



Closes: #386 (for real this time)

This PR open the share modal at the end of the voting workflow, to show a success message, as well as some buttons to share the vote.

This PR introduces a new customizable ShareModal (similar to TransactionProgress)

### How to test

1. Vote on an onchain proposal
2. On transaction confirmed, it should close the transaction modal, and open the share modal, with a button to view the tx, and button to share the vote on various social networks
3. The share message should contain the voted choice, as well as a link to the proposal (in case of shutter proposal, it should not contain the choice)
4. On offchain proposal, the share modal should not have the "View transaction" button

### To-Do

- [x] Merge #633 
- [x] Mege #627
